django_project/feti/templates/feti/pdf_template.html

```diff
@@ -12,6 +12,17 @@ <h2>Search : {{ query }} in
         all courses.
     {% endif %}
     </h2>
+
+    <a href="{{ url }}" target="_blank">Website</a>
```

Comment:
please don't use django context, instead generate it in javascript
using : `window.location.hostname`, and add necessary parameters from context